udp_driver/CMakeLists.txt

```
@@ -42,11 +50,16 @@ include_directories(include)
 ament_auto_add_library(${PROJECT_NAME} SHARED
   src/udp_socket.cpp
   src/udp_driver.cpp
-  examples/udp_driver_node.cpp)
+  src/udp_driver_node.cpp)
 ament_target_dependencies(${PROJECT_NAME} "Boost")
 
 target_compile_options(${PROJECT_NAME} PUBLIC "-O0" PRIVATE -Wno-sign-conversion -Wno-conversion)
```

for consistency with `io_context`, should this be changed to
```suggestion
target_compile_options(${PROJECT_NAME} PUBLIC "-O0" PRIVATE -Wall
```

---

This line shouldn't be required at all because `CMAKE_CXX_FLAGS` should cover that above. Removed.